File: src/beginner/day07/Question03.java
package beginner.day07;

public class Question03 {

    public static class Solution {

        public int solution(int n, int k) {
            int answer = 0;

            if (n >= 10) {
                k -= (n / 10);
            }

            answer = (n * 12000) + (k * 2000);

            return answer;
        }
    }

    public static void main(String[] args) {

        Solution solution = new Solution();

        System.out.println(solution.solution(10, 3));
        System.out.println(solution.solution(64, 6));
    }
}

File: src/beginner/day19/Question03.kt
package beginner.day19

class Question03{

    fun solution(array: IntArray, n: Int): Int {
        var answer: Int = 0

        for (i in array) {
            if (n == i) answer++
        }

        return answer
    }
}

fun main() {

    val solution = Question03()

    val array1 = intArrayOf(1, 1, 2, 3, 4, 5)
    val array2 = intArrayOf(0, 2, 3, 4)

    println(solution.solution(array1, 1))
    println(solution.solution(array2, 1))
}